mlir/include/mlir/Dialect/SCF/IR/SCFOps.td

```diff
@@ -846,8 +850,11 @@ def ParallelOp : SCF_Op<"parallel",
   ];
 
   let extraClassDeclaration = [{
-    ValueRange getInductionVars() {
-      return getBody()->getArguments();
+    // Get induction variables.
+    SmallVector<Value> getInductionVars() {
+      auto maybeInductionVars = getLoopInductionVars();;
```

Comment:
double `;`

Reply:
oh weird. i didn't know that even compiles